app/src/main/java/husaynhakeem/io/facedetectorapp/MainActivity.kt

```diff
@@ -1,22 +1,39 @@
 package husaynhakeem.io.facedetectorapp
 
 import android.os.Bundle
+import android.util.Log
 import android.util.Size
 import androidx.appcompat.app.AppCompatActivity
 import com.otaliastudios.cameraview.Facing
+import husaynhakeem.io.facedetector.FaceBounds
 import husaynhakeem.io.facedetector.FaceDetector
 import husaynhakeem.io.facedetector.Frame
 import husaynhakeem.io.facedetector.LensFacing
 import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
 
+    private val onFaceDetectionResultListener = object : FaceDetector.OnFaceDetectionResultListener{
+        override fun onFailure(exception: Exception) {
+            super.onFailure(exception)
+            exception.printStackTrace()
+            Log.e(TAG, "error ${exception.message}")
+        }
+
+        override fun onSuccess(faceBounds: List<FaceBounds>) {
+            super.onSuccess(faceBounds)
+            Log.e(TAG,"total faces ${faceBounds.size}")
```

Comment:
This isn't an error, it shouldn't use `Log.e`, instead an info or debug log should be used.

Also, please make sure to format the file after you're done (a space is missing before the log message).